muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/containers/workspace.tsx

```diff
@@ -71,43 +111,64 @@ export default class Workspace extends React.Component<WorkspaceProps, Workspace
     this.onHashChange = this.onHashChange.bind(this);
     this.renderWorkspaceHome = this.renderWorkspaceHome.bind(this);
     this.renderWorkspaceHelp = this.renderWorkspaceHelp.bind(this);
-    this.renderWorkspaceDiscussions = this.renderWorkspaceDiscussions.bind(this);
-    this.renderWorkspaceAnnouncements = this.renderWorkspaceAnnouncements.bind(this);
+    this.renderWorkspaceDiscussions =
+      this.renderWorkspaceDiscussions.bind(this);
+    this.renderWorkspaceAnnouncements =
+      this.renderWorkspaceAnnouncements.bind(this);
     this.renderWorkspaceAnnouncer = this.renderWorkspaceAnnouncer.bind(this);
     this.renderWorkspaceMaterials = this.renderWorkspaceMaterials.bind(this);
     this.renderWorkspaceUsers = this.renderWorkspaceUsers.bind(this);
     this.renderWorkspaceJournal = this.renderWorkspaceJournal.bind(this);
     this.renderWorkspaceManagement = this.renderWorkspaceManagement.bind(this);
-    this.renderWorkspacePermissions = this.renderWorkspacePermissions.bind(this);
-    this.loadWorkspaceDiscussionData = this.loadWorkspaceDiscussionData.bind(this);
-    this.loadWorkspaceAnnouncementsData = this.loadWorkspaceAnnouncementsData.bind(this);
-    this.loadWorkspaceAnnouncerData = this.loadWorkspaceAnnouncerData.bind(this);
-    this.loadWorkspaceMaterialsData = this.loadWorkspaceMaterialsData.bind(this);
+    this.renderWorkspaceEvaluation = this.renderWorkspaceEvaluation.bind(this);
+    this.renderWorkspacePermissions =
+      this.renderWorkspacePermissions.bind(this);
+    this.loadWorkspaceDiscussionData =
+      this.loadWorkspaceDiscussionData.bind(this);
+    this.loadWorkspaceAnnouncementsData =
+      this.loadWorkspaceAnnouncementsData.bind(this);
+    this.loadWorkspaceAnnouncerData =
+      this.loadWorkspaceAnnouncerData.bind(this);
+    this.loadWorkspaceMaterialsData =
+      this.loadWorkspaceMaterialsData.bind(this);
     this.loadWorkspaceHelpData = this.loadWorkspaceHelpData.bind(this);
     this.closeEnrollmentDialog = this.closeEnrollmentDialog.bind(this);
     this.closeSignupDialog = this.closeSignupDialog.bind(this);
 
-    this.onWorkspaceMaterialsBodyActiveNodeIdChange = this.onWorkspaceMaterialsBodyActiveNodeIdChange.bind(this);
-    this.onWorkspaceHelpBodyActiveNodeIdChange = this.onWorkspaceHelpBodyActiveNodeIdChange.bind(this);
+    this.onWorkspaceMaterialsBodyActiveNodeIdChange =
+      this.onWorkspaceMaterialsBodyActiveNodeIdChange.bind(this);
+    this.onWorkspaceHelpBodyActiveNodeIdChange =
+      this.onWorkspaceHelpBodyActiveNodeIdChange.bind(this);
 
     window.addEventListener("hashchange", this.onHashChange.bind(this));
 
     this.state = {
       enrollmentDialogOpen: !props.store.getState().status.loggedIn,
       signupDialogOpen: false,
-    }
+    };
   }
 
+  /**
+   * closeEnrollmentDialog
```

Comment:
What are these for?

The comment is literally the function name.

Reply:
Those "comments/documentations" are there now for placeholders only. I haven't look into those function that precisely to document what they are for. I added those just so code is more readable. Previously all functions where one big block of code without clear line breaks. And I think personally that there should be some base for documentation.